src/js/themes/base.js

```diff
@@ -1683,6 +1684,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         play: Play,
         reduceVolume: VolumeLow,
         volume: Volume,
+        description: AssistListening,
```

Comment:
It seems like there is a standard icon for Audio Description -- should we add that to grommet-icons to be able to use here?
<img width="1680" alt="Screen Shot 2022-04-20 at 10 53 20 AM" src="https://user-images.githubusercontent.com/12522275/164292736-9062c26c-eafa-4d6e-8004-0439e37f0bef.png">

Reply:
yes this icon would be preferred. what is the timeline/process for getting this added to grommet icons?

Reply:
I would create an issue for this on [grommet-icons](https://github.com/grommet/grommet-icons/issues) and add it to the backlog